kernel/src/path.rs

```diff
@@ -229,6 +231,20 @@ impl ParsedLogPath<Url> {
         }
         Ok(path)
     }
+
+    // TODO: remove after support for writing CRC files
+    #[allow(unused)]
+    /// Create a new ParsedCommitPath<Url> for a new CRC file
+    pub(crate) fn new_crc(table_root: &Url, version: Version) -> DeltaResult<Self> {
+        let filename = format!("{:020}.crc", version);
+        let path = Self::create_path(table_root, filename)?;
+        if path.file_type != LogPathFileType::Crc {
+            return Err(Error::internal_error(
+                "ParsedLogPath::new_crc created a non-crc path",
+            ));
+        }
```

Comment:
Shouldn't this be a `debug_assert`?

Reply:
yea it probably should - considering this is similar to what we do in the other spots i'll just take a follow up to fix them